.github/workflows/browsers-local.yml

```diff
@@ -0,0 +1,125 @@
+name: Browsers - local
+
+on:
+  pull_request:
+
+env:
+  NODE_VERSION: 20.x
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: ${{ matrix.MIGRATE_VERSION }} (Chrome/Firefox)
+    strategy:
+      fail-fast: false
+      matrix:
+        MIGRATE_VERSION: ["min"]
+        include:
+          - MIGRATE_VERSION: "esmodules"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Cache
+        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ env.NODE_VERSION }}-npm-lock-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ env.NODE_VERSION }}-npm-lock-
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run tests
+        run: |
+          npm run test:browser -- \
+            --jquery-migrate ${{ matrix.MIGRATE_VERSION }} \
+            --jquery git --jquery git.min --jquery git.slim --jquery git.slim.min \
+            --jquery 3.x-git --jquery 3.x-git.min --jquery 3.x-git.slim --jquery 3.x-git.slim.min \
+            --jquery 3.7.1 --jquery 3.7.1.slim \
+            --jquery 3.6.4 --jquery 3.5.1 --jquery 3.4.1 \
+            --jquery 3.3.1 --jquery 3.2.1 --jquery 3.1.1 --jquery 3.0.0 \
+            --retries 3 --hard-retries 1
+
+  ie:
+    runs-on: windows-latest
+    name: ${{ matrix.MIGRATE_VERSION }} (Edge in IE mode)
+    strategy:
+      fail-fast: false
+      matrix:
+        MIGRATE_VERSION: [ "min" ]
```

Comment:
No need for matrix. If you want to set it in one place, use env instead.

Reply:
Hmm, but we do also test `esmodules`? Doesn't that conflict with each other?

Reply:
We only test `esmodules` in the ubuntu job.

Reply:
Changed